services/cinema_hall.py

```diff
@@ -0,0 +1,17 @@
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> CinemaHall:
+    return CinemaHall.objects.all()
+
+
+def create_cinema_hall(
+    hall_name: str,
+    hall_rows: int,
+    hall_seats_in_row: int
+) -> CinemaHall:
+    return CinemaHall.objects.create(

```

Comment:
You don’t need to `return` anything in create/update/delete methods, check everywhere